### PR TITLE
Ot

### PR DESCRIPTION
**Pull request by markya0616**
**Before submitting a pull request**, please make sure the following is done:

### Checklist

- [ ] Fork [the repository](https://github.com/getamis/alice) and create your new branch from `master`.
- [ ] Please mention the name of community in Pull Request title.
- [ ] Git Commit Messages
    - [ ] Use the present tense (Also in Pull Request title): "Add feature" not "Added feature"
    - [ ] Use the imperative mood (Also in Pull Request title): "Move cursor to..." not "Moves cursor to..."
    - [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.

### CODE REVIEW

File: crypto/binaryfield/binary_field_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
+// Copyright © 2020 AMIS Technologies
```
Comment (cychuang0924) [inline crypto/binaryfield/binary_field_test.go:1]: 2020 -> 2021 My fault

File: crypto/ot/ot_ext_receiver.go [hunk 2]
```diff
@@ -0,0 +1,112 @@
+// Copyright © 2020 AMIS Technologies
```
Comment (cychuang0924) [inline crypto/ot/ot_ext_receiver.go:1]: 2020 -> 2021

File: crypto/ot/ot_ext_sender.go [hunk 3]
```diff
@@ -0,0 +1,316 @@
+// Copyright © 2020 AMIS Technologies
```
Comment (cychuang0924) [inline crypto/ot/ot_ext_sender.go:1]: 2020. ->2021

File: crypto/ot/ot_ext_sender.go [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+		tempfieldelement = scalMulFieldElement(chi[i], getRow(i, R))
+		vfieldelement, err = binaryfield.AddVector(vfieldelement, tempfieldelement)
+		if err != nil {
+			return nil, nil, err
+		}
+	}
+	return ufieldelement, vfieldelement, nil
+}
+
+func scalMulFieldElement(element *binaryfield.FieldElement, input []uint8) []*binaryfield.FieldElement {
```
Comment (cychuang0924) [inline crypto/ot/ot_ext_sender.go:281]: Do we move this function to the binary field?

Comment (markya0616) [reply crypto/ot/ot_ext_sender.go:281]: OK

**Review by cychuang0924 — APPROVED** [review]

**Pull request merged**